swap/src/main.rs

```diff
@@ -138,7 +138,7 @@ async fn main() -> Result<()> {
                 alice_addr,
                 alice_peer_id,
             );
-            let (swap, event_loop) = bob_factory.new_swap(swap_amounts).await?;
+            let (swap, event_loop) = bob_factory.new_swap_as_bob(swap_amounts).await?;
```

Comment:
This was just renamed in the previous patch, probably something to `fixup!` here :)